docs/feedback/README.md

```
@@ -0,0 +1,98 @@
+| T-3 | Cassandra keyspace 수동 생성 필요 | Infra | LOW | 인지 |
+| T-4 | Cloudflare Flexible SSL — 서버↔CF 구간 HTTP | Security | LOW | 인지 |
+| T-5 | Elastic IP 미사용 — 인스턴스 재시작 시 IP 변경 | Infra | MEDIUM | 미착수 |
+| T-6 | Redis 실제 미사용 — 설정만 있고 캐싱/Pub-Sub 적용 없음 | Infra | LOW | 인지 |
+| T-7 | WebSocket Simple Broker — 스케일아웃 시 Redis Pub/Sub 전환 필요 | Infra | LOW | 인지 |
+| T-8 | 메시지 카운터 ConcurrentHashMap — 서버 재시작 시 초기화 | Data | LOW | 인지 |
+| T-9 | JaCoCo 커버리지 40% (임시) — 50%로 복원 필요 | Quality | LOW | 인지 |
+| T-10 | 프론트엔드 에셋 수동 업로드 — gitignore + scp | DX | LOW | 인지 |
+
+### 현재 아키텍처 한계
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)